cli/config/config.go

```diff
@@ -118,3 +111,16 @@ func Load(configDir string) (*configfile.ConfigFile, error) {
 	}
 	return &configFile, nil
 }
+
+// LoadDefaultConfigFile attempts to load the default config file and returns
+// an initialized ConfigFile struct if none is found.
+func LoadDefaultConfigFile(err io.Writer) *configfile.ConfigFile {
+	configFile, e := Load(Dir())
+	if e != nil {
+		fmt.Fprintf(err, "WARNING: Error loading config file:%v\n", e)
```

Comment:
I realize this code is only moved, but it looks like a space is missing between `:` and `%v`.